typesafe/shared/src/main/scala/zio/config/typesafe/TypesafePropertyType.scala

```diff
@@ -0,0 +1,27 @@
+package zio.config.typesafe
+
+import com.github.ghik.silencer.silent
+import com.typesafe.config._
+import zio.config._
+
+import scala.util.Try
+
+@silent("Unused import")
+object TypesafePropertyType {
```

Comment:
Could we rename to ConfigValuePropertyType?

Reply:
@afsalthaj but the inner object is already `ConfigValuePropertyType`. We could drop the outer `TypesafePropertyType`.

Reply:
Makes sense.